x-pack/test/alerting_api_integration/spaces_only/tests/alerting/event_log.ts

```diff
@@ -589,6 +575,7 @@ export function validateEvent(event: IValidatedEvent, params: ValidateEventLogPa
   }
 
   const duration = event?.event?.duration;
+  const timestamp = Date.parse(event?.['@timestamp'] || 'udnefined');
```

Comment:
`undefined` instead of `udnefined`

Reply:
Good catch! Fixed in https://github.com/elastic/kibana/pull/119761/commits/81444485cd7dfc2297f3763a6b39f85c47b97b18